oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/util/LoggingDocumentStoreWrapper.java

```diff
@@ -283,6 +283,7 @@ public T call() throws Exception {
     }
 
     @Override
+    @NotNull
```

Comment:
is this change needed?

Reply:
IntelliJ warns about this missing `@NotNull` annotation since the interface declared this method `@NotNull`.

Reply:
it seems to be a preexisting issue though, unrelated to fullGC? I'd prefer to remove it.